src/dropdown/list/dropdown-list.component.ts

```diff
@@ -409,4 +411,10 @@ export class DropdownList implements AbstractDropdownView, AfterViewInit, OnChan
 		this.listElementList[index].classList.remove("bx--list-box__menu-item--highlighted");
 		this.listElementList[index].tabIndex = -1;
 	}
+
+	resetSelected() {
+		this.items = this.items.map((item) => {
```

Comment:
Probably want to do 
```ts
const clearedItems = this.getListItems().map(item => Object.assign({}, item, {selected: false}));
this.updateList(clearedItems);
```

`this.items` could return an observable, so we have `getListItems` which will always return the resolved `ListItem[]`. `updateList` also handles a few housekeeping tasks we need to do when a list is updated, so it's best to route everything through there.